jupyterhub_traefik_proxy/etcd.py

```diff
@@ -148,7 +148,7 @@ async def _kv_atomic_delete_route_parts(self, jupyterhub_routespec, route_keys):
         value = await maybe_future(self._etcd_get(jupyterhub_routespec))
         if value is None:
             self.log.warning("Route %s doesn't exist. Nothing to delete", jupyterhub_routespec)
-            return
+            return None, None
```

Comment:
Deleting a route that doesn't exist, shouldn't fail.
```suggestion
            return True, None
```
It would be super awesome if you could extend this check to consul too to avoid performing an unnecessary transaction (for the empty string).

Because both the empty string and None evaluate to False, both consul and etcd could have this check:
```python
if value:
   return True, None
```